contracts/voting/dao-voting-cw721-staked/src/state.rs

```diff
@@ -39,7 +39,7 @@ pub const TOTAL_STAKED_NFTS: SnapshotItem<Uint128> = SnapshotItem::new(
 );
 
 /// The maximum number of claims that may be outstanding.
-pub const MAX_CLAIMS: u64 = 100;
+pub const MAX_CLAIMS: u64 = 70;
```

Comment:
What do you think we should lower this to?

Here is some data with different values of MAX_CLAIMS:
```
* 100:
    * ERROR
    * gas_used: 57793
    * gas_wanted: 12131747

* 95:
    * ERROR
    * gas_used: 57793
    * gas_wanted: 11533653

* 90: 
    * ERROR
    * gas_used: 57793
    * gas_wanted: 10935558

* 85:
    * ERROR
    * gas_used: 57793
    * gas_wanted: 10337451
   
* 80: 
    * WORKS
    * gas_used: 57793
    * gas_wanted: 9739358

* 70: 
    * WORKS
    * gas_used: 57753
    * gas_wanted: 8543711
```

Reply:
this feels like a very complex expected value calculation that i lack the ability to make. on one hand, bad if NFTs are stuck temporarily in the contract, on the other hand _very_ bad if Juno lowers the block gas limit again and then NFTs are stuck until we can do an upgrade..

i think 70 sounds reasonable. do you know why Juno lowered the block gas limit? knowing that might help us evaluate `P(lower gas limit in the future)`.

Reply:
Im having a hard time finding more info on why they lowered it. I assume it was for security reasons. There was probably a contract doing something really computationally expensive that was either causing a small halt or maybe threatening to cause a halt.

Reply:
The other thing to keep in mind when we are talking about gas price estimation for these txs (not the actual gas_used that comes back from the tx response), is that we are using some gas price constants that are [bit larger than what most people use](https://github.com/DA0-DA0/dao-contracts/blob/main/ci/configs/cosm-orc/ci.yaml#L6-L7). So we have more headroom for that reason too. 

It's hard to find consistent values for these to compare, but it looks like most people use  a `gas_adjustment` from 1.0 -> 1.5 (we are using 1.3 in cosmjs). And it looks like we are using the [min gas price in dao dao ui](https://github.com/DA0-DA0/dao-dao-ui/blob/development/packages/stateful/components/WalletProvider.tsx#L79), but the recommended 'high' from the [chain registry](https://github.com/cosmos/chain-registry/blob/master/juno/chain.json#L18-L21) is half of what we are using in the integration tests.

Reply:
The other thing to keep in mind is that when we are talking about our gas simulation as opposed to actual gas 
usage, is that we are using a `gas_adjustment` that is a bit higher that what most people use. 

[Ours is 1.5](https://github.com/DA0-DA0/dao-contracts/blob/main/ci/configs/cosm-orc/ci.yaml#L7). It's hard to find consistent values for these to compare, but it looks like most people use  a `gas_adjustment` from 1.0 -> 1.5 (we are using [1.3 in cosmjs for example](https://github.com/cosmos/cosmjs/blob/main/packages/stargate/src/signingstargateclient.ts#L286) in the DAO DAO UI). 

So we have a bit more headroom for that reason too.

Reply:
> So we have a bit more headroom for that reason too

nice :)

> I assume it was for security reasons.

this is my assumption as well. based on that, i think it is OK to make a "leap of faith" here that the new value was tested to be safe and that we don't need to lower again.

i'm good with `70` :)